Library/Homebrew/cask/spec/cask/artifact/uninstall_spec.rb

```diff
@@ -7,12 +7,24 @@
     Hbc::Artifact::Uninstall.new(cask, command: Hbc::FakeSystemCommand)
   }
 
+  let(:test_home_path) { TEST_FIXTURE_DIR }
+  let(:original_home_path) { ENV['HOME'] }
+  let(:absolute_path) { TEST_FIXTURE_DIR/'cask'/'absolute_path' }
+  let(:path_with_tilde) { TEST_FIXTURE_DIR/'cask'/'path_with_tilde' }
+  let(:glob_path1) { TEST_FIXTURE_DIR/'cask'/'glob_path1' }
+  let(:glob_path2) { TEST_FIXTURE_DIR/'cask'/'glob_path2' }
+
   before(:each) do
+    ENV['HOME'] = test_home_path
```

Comment:
It has in https://github.com/Homebrew/brew/pull/2042/files#diff-2c3341e5f3e51c885e64c56769cbc57bR12 and other fixture casks. But during zap or delete it expands so in test I compare it with the full path like `TEST_FIXTURE_DIR/'cask'/'path_with_tilde'`. I can use `Pathname.new('~/cask/path_with_tilde').expand_path` instead but that's like test function that should return `6` against `Math::sqrt(36)` instead of `6`.